src/components/RepoBadge/RepoBadge.js

```diff
@@ -52,15 +53,20 @@ const RepoBadge = ({ repo }) => {
         <AppIcon
           app={{ baseUrl, icons }}
           src={knownIcon || undefined}
-          radius={3}
-          css="min-width: 22px;"
+          radius={RADIUS}
+          css={`
+            min-width: 22px;
```

Comment:
I don't think we need to set the `min-width` here, but if we do we should probably set it to 3 GU

Reply:
Fixed [here](https://github.com/aragon/aragon/pull/936/commits/ea72d4ad5ca700ba6728d7cd601fdefd33e62808)